notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>I suggest the following for the Dataset section:</p><p> </p><p>This tutorial uses Synthetic_2 (Syn2) data, described in Section 4.1 of the&nbsp;<a href="https://arxiv.org/pdf/1802.07814.pdf" target="_blank">Learning to Explain</a>&nbsp;paper. The input feature X is generated from a 10-dimensional standard Gaussian. The response variable Y is generated from feature X[3:6] only. The data has been split into training and prediction sets and has been uploaded to Google Cloud Storage:</p><ul><li>Training data: gs://cloud-samples-data/ai-platform-unified/datasets/tabnet/tab_net_input/syn2_train.csv.</li><li>Prediction data: gs://cloud-samples-data/ai-platform-unified/datasets/tabnet/tab_net_output/syn2</li></ul><p> </p><p>At this time, the TabNet model file is not publicly available.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. Thanks.</p>